### PR TITLE
feat: added landing

### PR DESCRIPTION
**Pull request by eugenederii**

### CODE REVIEW

File: src/modules/landing/components/reviews-card/review-cards.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import Image from 'next/image';
+// components
+import { Typography } from '@/modules/core/components/typogrpahy';
+// types
+import type { ReviewCardProps } from './review-cards.interface';
+import img from '@/assets/images/girl.png';
+
+export const ReviewCard: FC<ReviewCardProps> = ({
+  text,
+
```
Comment (IhorBalan) [inline src/modules/landing/components/reviews-card/review-cards.tsx:11]: Remove space here

File: src/modules/landing/components/reviews-card/review-cards.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  name,
+  occupation,
+}) => {
+  return (
+    <div className=' p-4 border-2 rounded-[12px] border-[#F9F9F9] flex flex-col items-center xl:flex-row gap-6 '>
```
Comment (IhorBalan) [inline src/modules/landing/components/reviews-card/review-cards.tsx:16]: There a lot of unnecessary spaces at the start and end of the string, please remove them in whole file. And make a double check in other files.

File: src/modules/landing/landing-image/facebook.png
Comment (IhorBalan) [inline src/modules/landing/landing-image/facebook.png:1]: It would be nice to have svg icons.

File: src/modules/landing/landing-image/applePay.png
Comment (IhorBalan) [inline src/modules/landing/landing-image/applePay.png:1]: For the apple better to use icons that we have in their guideline https://developer.apple.com/app-store/marketing/guidelines/

File: src/modules/landing/landing-image/googlePay.png
Comment (IhorBalan) [inline src/modules/landing/landing-image/googlePay.png:1]: For google better to use that they had in their guideline https://play.google.com/intl/en_us/badges/

**Review by IhorBalan — CHANGES_REQUESTED** [review]

**Comment by VadimTarasenko** [comment]
Screenshot 2024-05-22 at 01 13 06

- Remove header shadow.
- Initial section should fit window height and centered (second section shouldn't be visible when scroll on top). @eugenederii

**Comment by VadimTarasenko** [comment]
Screenshot 2024-05-22 at 01 16 23

Use svg for this links. On hover change color to white.

**Comment by VadimTarasenko** [comment]
Screenshot 2024-05-22 at 01 19 34

Update this buttons in according this article: https://ezedi.medium.com/how-to-get-original-googles-playstore-and-apple-s-appstore-button-for-your-app-or-website-ecbe2c66f0aa

**Comment by VadimTarasenko** [comment]
Screenshot 2024-05-24 at 16 34 53

When we scroll to the left, slider items shouldn't be croped

**Comment by VadimTarasenko** [comment]
Screenshot 2024-06-03 at 13 13 10

Slides should have the same height

**Comment by VadimTarasenko** [comment]
Screenshot 2024-06-05 at 13 15 34

Improve quality of this images. Export them separately from figma.

**Comment by VadimTarasenko** [comment]
image

GooglePlay and Appstore buttons overlap header @eugenederii

**Review by VadimTarasenko — APPROVED** [review]

**Pull request merged**